seed/seed20.go

```diff
@@ -100,55 +100,54 @@ type seed20 struct {
 func (s *seed20) OptionalContainers() (OptionalContainers, error) {
 	model := s.Model()
 
-	optionalSnaps := make(map[string]bool)
-	optionalComponents := make(map[string]map[string]bool)
+	requiredSnapsInModel := make(map[string]bool)
+	requiredComponentsInModel := make(map[string]map[string]bool)
 	for _, snap := range model.AllSnaps() {
-		optionalComps := make(map[string]bool)
-		for compName, comp := range snap.Components {
-			if comp.Presence == "optional" {
-				optionalComps[compName] = true
-			}
+		if snap.Presence == "required" || snap.Presence == "" {
```

Comment:
can Presence actually be "" ? the code in asserts/model.go seem to hint that it can but iwith the code itself there it doesn't seem to be the case?

Reply:
Ah no it actually can't. I'll remove that part of the condition. I was thinking it was empty for essential snaps, but that is only in the assertion itself. When we assemble the assertion, we default it to required.